melodic/distribution.yaml

```diff
@@ -3405,6 +3405,17 @@ repositories:
       url: https://github.com/ros/geometry_tutorials.git
       version: indigo-devel
     status: maintained
+  geos_cmake_module:
```

Comment:
@daniel-dsouza no `doc` entry? Just checking.

Reply:
Sorry, I don't have a site for documentation, and do not know the proper way to include the README.MD in the source repo.

Reply:
@daniel-dsouza  `doc` entries are for package indexing and documentation generation (see [here](http://wiki.ros.org/rosdistro/Tutorials/Indexing%20Your%20ROS%20Repository%20for%20Documentation%20Generation)). But based on your comment. this is likely irrelevant for you right now.